src/components/menus/Items.ts

```diff
@@ -111,9 +112,80 @@ export class Items<TGameStartr extends FullScreenPokemon> extends Component<TGam
      * @param settings   Custom attributes to apply to the menu.
      */
     public openItemMenu(listing: IInventoryListing, settings: IItemMenuSettings): void {
-        const options: any[] = [
+        let options: any[] = [];
+        if (this.gameStarter.features.heldItems) {
+            options = [
```

Comment:
This is a lot of duplicated code. I really dislike duplicated code.

Just `.push` or `unshift` the extra members you need. DRY.